docs/products/compute/compute-instances/guides/placement-groups/index.md

```diff
@@ -0,0 +1,213 @@
+
+Because of the issue with the API, I'm including the Availability section in the docs. It's easy enough to update the docs on a weekly release cadence as we add new supported regions. (Do we anticipate qualifying regions quicker than once a week?)
+
+-->
+
+The Placement Groups service is available in select regions. Currently, this includes:
+
+- Miami, FL (us-mia)
+
+- Chicago, IL (us-ord)
+
+{{< note >}}
+Placement Groups is in limited availability. Throughout this phase, we expect to increase the number of supported regions.
+{{< /note >}}
+
+## Affinity, enforcement, and compliance
+
+To distribute your compute instances in a placement group to meet your model, we use the industry-recognized affinity standard. You can pick from one of two types:
+
+- **Affinity**. Your compute instances are physically close together, possibly on the same host. Set this as your preferred container to help with performance.
+
+- **Anti-affinity**. Your compute instances are placed in separate fault domains, but they're still in the same region. Use this preferred container to better support a high-availability model.
+
+You also select how your affinity type should be enforced, when adding more compute instances to it in the future. This determines whether or not the placement group remains accessible – is “compliant”-- after you add a new compute instance:
+
+- **Strict (Best practice)**. You can't add more compute instances to your placement group if your preferred container lacks capacity or is unavailable. For example, let's assume you've set **Anti-affinity** as your affinity type. If you try to add a compute instance that's on the same host, or there's no capacity outside that host in the region, you get an error and can't add the compute instance. This helps you keep your placement group compliant, because you can only pick compute instances that fit your desired model.
+
+- **Flexible**. You can add more compute instances to your placement group even if they're outside your affinity type's preferred container. However, if you add one and it violates your affinity type, the placement group becomes non-compliant. Once the necessary capacity is available in the data center, we physically move the compute instance for you, to fit your affinity type's preferred container and make it compliant again. This can work for you if you know you need to add more compute instances to your group in the future.
+
+### Non-compliance precedence
```

Comment:
Suggestion for entire section:

> ### Fixing Non-Compliance
> 
> If a placement group becomes non-compliant, Akamai staff is alerted and will bring it back into compliance as quickly as possible. Non-compliance can only be fixed by Akamai staff; customers are not able to fix this themselves. 
>
> By design, a strict placement group cannot be made non-compliant during normal operation (such as when creating it or managing the Compute Instances that belong to the group). In rare cases, non-compliance can occur during maintenance, such as when an instance needs to be migrated to perform maintenance on a host machine. Because of this, fixing non-compliance for strict placement groups is prioritized over flexible groups.